.github/workflows/coverage_pr.yml

```diff
@@ -106,6 +106,8 @@ jobs:
         if: ${{ steps.ut-pr.outputs.paths != '' }}
         run: |
           cd $GITHUB_WORKSPACE/matrixone
+          ps aux | grep 'mo-service' | grep -v 'grep' | awk '{print $2}' | xargs kill -s SIGINT
+          while(($(ps aux|grep 'mo-service' | grep -v 'grep' | awk '{print $2}'))); do echo 'mo-service kill not finished'; sleep 1s; done
```

Comment:
better to add a timeout or max retry time

Reply:
OK